cmd/k8s-operator/ingress-for-pg.go

```diff
@@ -247,17 +245,18 @@ func (a *IngressPGReconciler) maybeProvision(ctx context.Context, hostname strin
 		tags = strings.Split(tstr, ",")
 	}
 
-	vipSVc := &VIPService{
+	vipSvc := &VIPService{
 		Name:    hostname,
 		Tags:    tags,
+		Ports:   []string{"443"}, // always 443 for Ingress
```

Comment:
Also am not setting ports here as we deployed the change that requires ports to be set